src/features/todo/index.tsx

```diff
@@ -74,3 +134,18 @@ export const TodoList = () => {
     </>
   );
 };
+
+const Chip = ({
+  children,
+  onClose,
+}: {
+  children: string;
+  onClose: () => void;
+}) => {
+  return (
+    <span className="flex items-center bg-white text-stone-700 text-xs px-2 py-1 rounded-full">
```

Comment:
(Just a suggestion) What do you think about these styles to adjust it a bit to the rest of the components beside?
```suggestion
    <span className="flex items-center font-bold bg-white text-stone-700 text-xs px-3 py-2 rounded-full">
```
<img width="479" alt="Screenshot 2025-03-03 at 18 21 05" src="https://github.com/user-attachments/assets/521c94f1-7075-43de-b6b7-8dd4ee0eb5bf" />